packages/core/helper-plugin/src/features/RBAC.tsx

```diff
@@ -1,36 +1,25 @@
 import * as React from 'react';
 
-/**
- * @preserve
- * @typedef {Object} Permission
- * @property {string} action
- * @property {unknown[]} conditions
- * @property {number} id
- * @property {Record<string, unknown>} properties
- * @property {string} subject
- */
+interface Permission {
+  action: string;
+  conditions: unknown[];
+  id: number;
+  properties: Record<string, unknown>;
+  subject: string | null;
+}
 
-/**
- * @preserve
- * @typedef {import('react-query').QueryObserverBaseResult<Permission[]>['refetch']} RefetchPermissionsFn
- */
+type RefetchPermissionsFn = import('react-query').QueryObserverBaseResult<Permission[]>['refetch'];
```

Comment:
I dont think we've imported a type like this anywhere in the TS files 🤔 how come you went for this way?